packages/build-tools/src/common/setup.ts

```diff
@@ -59,7 +59,24 @@ export async function setupAsync<TJob extends BuildJob>(ctx: BuildContext<TJob>)
     await runInstallDependenciesAsync(ctx);
   });
 
+  await ctx.runBuildPhase(BuildPhase.READ_APP_CONFIG, async () => {
+    const appConfig = ctx.appConfig;
+    ctx.logger.info('Using app configuration:');
+    ctx.logger.info(JSON.stringify(appConfig, null, 2));
+    await validateAppConfigAsync(ctx, appConfig);
+  });
+
   if (ctx.job.triggeredBy === BuildTrigger.GIT_BASED_INTEGRATION) {
+    if (!ctx.appConfig.ios?.bundleIdentifier) {
```

Comment:
Can the GitHub build be triggered for only 1 platform? If so, wouldn't running an Android build always fail on this condition, and iOS build on the second?

Reply:
yeah, it can, makes sense

Reply:
fixed